src/Telegram.Bot/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Telegram.Bot.Tests.Integ")]
```

Comment:
This file could be removed. Our sys integ tests are meant to test the tg client the same way our users might work with it. That's not meant for unit testing.

Reply:
oh, I left it accidentally, will remove it.